classes/04class/exercises/c04-iac01/alanlima/ec2.tf

```diff
@@ -0,0 +1,19 @@
+locals {
```

Comment:
Why did you decide to use local var's here ?
And not the variables file 😄 ?

Reply:
Just playing around with the options 🙄
Initially, I was using locals because I was combining the `common_tags` with the variable `devops_class` (looks like that I can't do it in the `tfvars` file). After refactoring, I move the common_tags to variables and let those locals for ec2 there 😅

Reply:
Experimenting is the best!
When you begin your TF Module exercise you'll get to understand the power of the tfvars file and in combination with makefiles you run certain resources individually for example

terraform apply -target module.vpc  -var-file="var.tfvars" -auto-approve

Anywho,
Keep up the good work!

Reply:
Cool, I just had a glance at Terraform modules and I found it fabulous. I am looking forward for that.

Thanks for your review, I will keep the good work for sure 😁